airbyte-config/models/src/test/java/io/airbyte/config/helpers/RawListToStandardXDefinitionsTest.java

```diff
@@ -0,0 +1,160 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+public class RawListToStandardXDefinitionsTest {
+
+  private static final String goodDesDefYaml =
+      "- destinationDefinitionId: a625d593-bba5-4a1c-a53d-2d246268a816\n"
+          + "  name: Local JSON\n"
+          + "  dockerRepository: airbyte/destination-local-json\n"
+          + "  dockerImageTag: 0.1.4\n"
+          + "  documentationUrl: https://docs.airbyte.io/integrations/destinations/local-json";
+  private static final String duplicateId =
+      "- destinationDefinitionId: a625d593-bba5-4a1c-a53d-2d246268a816\n"
+          + "  name: Local JSON\n"
+          + "  dockerRepository: airbyte/destination-local-json\n"
+          + "  dockerImageTag: 0.1.4\n"
+          + "  documentationUrl: https://docs.airbyte.io/integrations/destinations/local-json"
+          + "- destinationDefinitionId: a625d593-bba5-4a1c-a53d-2d246268a816\n"
+          + "  name: JSON 2\n"
+          + "  dockerRepository: airbyte/destination-local-json\n"
+          + "  dockerImageTag: 0.1.4\n"
+          + "  documentationUrl: https://docs.airbyte.io/integrations/destinations/local-json";
+  private static final String duplicateName =
+      "- destinationDefinitionId: a625d593-bba5-4a1c-a53d-2d246268a816\n"
+          + "  name: Local JSON\n"
+          + "  dockerRepository: airbyte/destination-local-json\n"
+          + "  dockerImageTag: 0.1.4\n"
+          + "  documentationUrl: https://docs.airbyte.io/integrations/destinations/local-json\n"
+          + "- destinationDefinitionId: 8be1cf83-fde1-477f-a4ad-318d23c9f3c6\n"
+          + "  name: Local JSON\n"
+          + "  dockerRepository: airbyte/destination-csv\n"
+          + "  dockerImageTag: 0.1.8\n"
+          + "  documentationUrl: https://docs.airbyte.io/integrations/destinations/local-csv";
+  private static final String badData =
+      "- destinationDefinitionId: a625d593-bba5-4a1c-a53d-2d246268a816\n"
+          + "  name: Local JSON\n"
+          + "  dockerRepository: airbyte/destination-local-json\n"
+          + "  dockerImageTag: 0.1.8\n"
+          + "  documentationUrl";
+
+  @Nested
+  @DisplayName("vertifyAndConvertToJsonNode")
+  public class VerifyAndConvertToJsonNode {
+
+    private static final String ID_NAME = "destinationDefinitionId";
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Test
+    @DisplayName("should correctly read yaml file")
+    public void correctlyRead() throws JsonProcessingException {
+      var jsonDefs = RawListToStandardXDefinitions.verifyAndConvertToJsonNode(ID_NAME, goodDesDefYaml);
+      var defList = mapper.treeToValue(jsonDefs, StandardDestinationDefinition[].class);
+      assertEquals(1, defList.length);
+      assertEquals("Local JSON", defList[0].getName());
+    }
+
+    @Test
+    @DisplayName("should error out on duplicate id")
+    public void duplicateId() {
```

Comment:
by convention methods for tests should contain the word test as a prefix or suffix.